gbpservice/nfp/common/constants.py

```diff
@@ -65,6 +73,7 @@
     CUSTOM_JSON: 'custom_json'}
 
 LOADBALANCER_RPC_API_VERSION = "2.0"
+LOADBALANCERV2_RPC_API_VERSION = "1.0"
```

Comment:
Explained in the discussion. Agent manager for lbaasv1 and lbaasv2 are two different api.